src/AcmEmptyState/AcmEmptyState.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
```

Comment:
I re-implemented this from `AcmEmptyPage` to `AcmEmptyState` because I believe the other teams have some cases where an empty state doesn't occupy a full page